website/deploy/staging/snack.env

```
@@ -1,6 +1,7 @@
 SERVER_URL=https://staging.expo.dev
 API_SERVER_URL=https://staging.exp.host
 SENTRY_ENVIRONMENT=staging
+LEGACY_SNACK_SERVER_URL=https://staging.snack.expo.dev
```

Minor issue, this is not even used on the pods I think and I'm not sure even locally, but lets fix it

```suggestion
LEGACY_SNACK_SERVER_URL=https://staging.snack.expo.io
```

---

Feel free to merge 👍